### PR TITLE
Added TTL index upon init to perform telemetry data expiration

### PR DESCRIPTION
**Pull request by edtsoi430**

### CODE REVIEW

File: internal/jager-mongodb/reader.go [hunk 1]
```diff
@@ -225,6 +225,19 @@ func (s *SpanReader) findTraceIDs(ctx context.Context, query *spanstore.TraceQue
 		"$lt": query.StartTimeMax,
 	}
 
+	// Filtering by concatenation of tags.
```
Comment (dmichel1) [inline internal/jager-mongodb/reader.go]: isn't this already merged into the main branch? doing a rebase locally should pull those changes into your branch.

e.g. `git pull origin master --rebase`

File: internal/jager-mongodb/reader_test.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
+package jager_mongodb_test
```
Comment (dmichel1) [inline internal/jager-mongodb/reader_test.go]: can we leave the tests our of this PR until we have a framework to load test data?

**Review by dmichel1 — CHANGES_REQUESTED** [review]
2 comments than LGTM.

File: internal/jager-mongodb/reader_test.go (hunk 2 above)
Comment (edtsoi430) [reply internal/jager-mongodb/reader_test.go]: just removed it

**Review by dmichel1 — APPROVED** [review]

**Pull request merged**